integration_test/issue_wizard/ivido/ivido_demo_client.dart

```diff
@@ -0,0 +1,190 @@
+import 'package:irmamobile/src/screens/scanner/widgets/qr_scanner.dart';
+import 'package:irmamobile/src/widgets/collapsible.dart';
+import 'package:irmamobile/src/widgets/irma_card.dart';
+import 'package:irmamobile/src/widgets/irma_markdown.dart';
+import 'package:irmamobile/src/widgets/irma_stepper.dart';
+import 'package:irmamobile/src/widgets/issuer_verifier_header.dart';
+import 'package:irmamobile/src/widgets/session_progress_indicator.dart';
+
+import '../../helpers/helpers.dart';
+import '../../helpers/issuance_helpers.dart';
+import '../../irma_binding.dart';
+import '../../util.dart';
+
+Future<void> ividoDemoClientTest(WidgetTester tester, IntegrationTestIrmaBinding irmaBinding) async {
+  await pumpAndUnlockApp(tester, irmaBinding.repository);
+
+  // Navigate to the scanner screen
+  await tester.tapAndSettle(find.byKey(const Key('nav_button_scanner')));
+
+  // Create a pointer for the Ivido demo
+  final ividoIssueWizardPointer = IssueWizardPointer('irma-demo-requestors.ivido.demo-client');
+
+  // Feed the pointer to the onFound in the QrScanner widget to mock starting a session via QR code
+  final qrScannerFinder = find.byType(QRScanner);
+  final qrScannerWidget = qrScannerFinder.evaluate().first.widget as QRScanner;
+  qrScannerWidget.onFound(ividoIssueWizardPointer);
+
+  // This takes quite some time
+  await tester.pumpAndSettle(const Duration(seconds: 3));
+
+  // Expect the issue wizard
+  expect(find.byType(WizardScaffold), findsOneWidget);
+
+  // Expect to find the header
+  final headerFinder = find.byType(IssuerVerifierHeader);
+  expect(headerFinder, findsOneWidget);
+
+  // Expect the header with the right text
+  expect(
+    find.descendant(
+      of: headerFinder,
+      matching: find.text('Ivido PHE'),
+    ),
+    findsOneWidget,
+  );
+
+  // Expect the right background color from Ivido
+  final headerWidget = headerFinder.evaluate().first.widget as IssuerVerifierHeader;
+  expect(headerWidget.backgroundColor, const Color(0xffe7dffe));
+
+  // Test the questions and answers
+  final qAndAs = {
+    'Which data are added?':
+        'You retrieve your personal data. You need this data to show who you are. That way only you can log in on your account.',
+    'Where do you get the data?':
+        'You retrieve your personal data from the dutch personal records database ("Basis Registratie Personen", BRP). You log in with DigiD at the municipality of Nijmegen. Even if you live in another city. Nijmegen offers this service for everyone with a DigiD.',
+    'How does it work?':
+        'Your login data are stored only in your IRMA app on your phone. To log in at Ivido you show who you are with your IRMA app.',
+    'What is Ivido?':
+        'Ivido is a Personal Health Environment (PHE, dutch: PHO). In Ivido you can store everything about your healt. You choose yourself with who you share this data. You use IRMA to register and log in at Ivido.',
```

Comment:
~PHO~ PGO

Reply:
This is all part of the irma-demo-requestors scheme. 

https://github.com/privacybydesign/irma-demo-requestors/blob/3c39f28476ee579a29da008b003dc0e42fcc087d/requestors.json

Im just testing that what is described there.